sky/packages/sky/lib/src/widgets/editable_text.dart

```diff
@@ -129,7 +132,9 @@ class EditableString implements KeyboardClient {
     onUpdated();
   }
 
-  void submit(SubmitAction action) {}
+  void submit(SubmitAction action) {
+    onSubmitted();
```

Comment:
This was borrowed from the existing onUpdated, which isn't doing a check (probably under the assumption that EditableString would be used within Input, which provides an onUpdated handler)

Added assertions in the ctor for onUpdated and onSubmitted